packager/packager.cc

```diff
@@ -1661,8 +1649,6 @@ void populatePackagePathPrefixes(core::GlobalState &gs, ast::ParsedFile &package
 void populatePackageEdges(core::GlobalState &gs, ast::ParsedFile &package, PackageInfoImpl &info) {
     rewritePackageSpec(gs, package, info);
     for (auto &importedPackageName : info.importedPackageNames) {
-        populateMangledName(gs, importedPackageName.name);
-
```

Comment:
I think if you rebase on `master`, `populatePackageEdges` will turn into a single call to `rewritePackageSpec` after this change. Might be worth inlining it into the existing call sites.